code/game/objects/items/ego_weapons/teth.dm

```diff
@@ -118,3 +118,18 @@
 /obj/item/ego_weapon/wrist/melee_attack_chain(mob/user, atom/target, params)
 	..()
 	user.changeNext_move(CLICK_CD_MELEE * 0.30)
+
+/obj/item/ego_weapon/regret
+	name = "regret"
+	desc = "Before swinging this weapon, expressing one’s condolences for the demise of the inmate who couldn't even have a funeral would be nice."
+	special = "This weapon attacks extremely slowly."
+	icon_state = "regret"
+	force = 60					//Lots of damage, way less DPS
```

Comment:
Reduce to 40, that way it does 2x damage for 2x cooldown and not 3x damage for 2x cooldown

Reply:
done!